server/models/operation.js

```diff
@@ -22,6 +22,7 @@ let Operation = americano.getModel('bankoperation', {
     categoryId: String,
     attachments: Object,         // {linkTranslationKey: String, linkPlainEnglish: String, url: String}
     operationTypeID: String,
+    customLabel: String,
```

Comment:
I've just checked and the client side shouldn't need any special attention, only the server side will (server/controllers/operations.js)